src/Components/RequestList.tsx

```diff
@@ -392,7 +397,13 @@ export const ProceedingRow = (props: ProceedingRowProps) => {
                                     msg.subject && (
                                         <>
                                             <br />
-                                            <span className="icon icon-email">{msg.subject}</span>
+                                            <span
+                                                className={`icon ${iconClassForTransportMedium[msg.transport_medium]}`}
+                                                title={t('sent-via-medium', 'my-requests', {
+                                                    medium: t(msg.transport_medium, 'my-requests'),
+                                                })}>
```

Comment:
This should actually also be an `aria-label` (https://www.a11yproject.com/posts/title-attributes/).